src/helpers/loadOrCreatePlayer.ts

```diff
@@ -1,21 +1,28 @@
 import { Address, BigInt, log } from '@graphprotocol/graph-ts'
-import { Player } from '../../generated/schema'
+import {
+  Player,
+  DripTokenPlayer,
+  BalanceDripPlayer,
+} from '../../generated/schema'
 
-import { playerId } from './idTemplates'
+import {
+  playerId,
+  dripTokenPlayerId,
+  balanceDripPlayerId,
+} from './idTemplates'
 
 export function loadOrCreatePlayer(
-  prizePool: Address,
+  source: Address,
   player: Address
 ): Player {
-  const id = playerId(prizePool.toHex(), player.toHex())
+  const id = playerId(source.toHex(), player.toHex())
```

Comment:
Why change this from `prizePool` to `source`, it's always going to be attached to a prize pool, yeah?

Reply:
fixed!